### PR TITLE
DICT-57: Updates to the TREATMENT node - YERTLE

### PR DESCRIPTION
**Pull request by tlicht3**
For a complete list of changes refer to this ticket: https://jira.opensciencedatacloud.org/browse/DICT-57

### CODE REVIEW

File: gdcdictionary/schemas/treatment.yaml [hunk 1]
```diff
@@ -634,9 +666,12 @@ properties:
       - Radiation, Combination
       - Radiation, Cyberknife
       - Radiation, External Beam
+      - Radiation, Hypofractionated
       - Radiation, Implants
       - Radiation, Intensity-Modulated Radiotherapy
       - Radiation, Internal
+      - Radiation, Mixed Photon Beam
+      - Radiation, Photon Beam
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:674]: not sure if it matters, but _Radiation, Photon Beam_ shows up twice (lines 674 and 675)

File: gdcdictionary/schemas/treatment.yaml [hunk 2]
```diff
@@ -3417,10 +3854,11 @@ properties:
...
       - Oregovomab
+      - Orelabrutinib
       - Ormaplatin
       - Ortataxel
       - Orteronel
-      - OSI-774 (erlotinib; Tarceva)
+      - Orvacabtagene Autoleucel
       - Osilodrostat
       - Osimertinib
       - Other
```
Comment (wteouchicago) [inline gdcdictionary/schemas/treatment.yaml:3864]: Do we need to remove "Other" here (in addition to removing it from the deprecated_enum section)?

Comment (tlicht3) [reply gdcdictionary/schemas/treatment.yaml:3864]: @wteouchicago we are not removing "Other". Instead, we are removing it from the deprecated list, as we need it to stay on the list of values.

File: gdcdictionary/schemas/treatment.yaml (hunk 1 above)
Comment (tlicht3) [reply gdcdictionary/schemas/treatment.yaml:674]: @wteouchicago Proton and Photon are two different types of radiation, so we need both of them. Thanks for checking on this!

Comment (wteouchicago) [reply gdcdictionary/schemas/treatment.yaml:674]: Ah thanks! I clearly need my eyes checked!

**Review by wteouchicago — APPROVED** [review]

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**